buildkite.yaml

```diff
@@ -5,7 +5,7 @@ steps:
       - "npm test"
     plugins:
       - docker#v3.7.0:
-          image: node:latest
+          image: node:lts
```

Comment:
Hmm, is this image change and the node version bump essential for this PR?

Reply:
Oh, I see that you merged a branch in with these changes to get the tests to pass. Perhaps it would be good to call that out and merge the other branch before this one? Or perhaps rebase this branch onto the branch with the node update?

Reply:
@niceking Good point! I tested the integration with Test Analytics locally (i.e. running example tests and sending the results to local analytics API), and I can write documentation for that. But I think it also worth making the API URL configurable to easily debug/test the integration locally; similar to what we did for Ruby collector)

Reply:
@niceking as for the `node` update, I'll make sure to merge the other one first and rebase this branch. I'll put this PR back to draft.